lib/chef/knife/acl_remove.rb

```diff
@@ -19,45 +19,27 @@
 module OpscodeAcl
   class AclRemove < Chef::Knife
     category "OPSCODE HOSTED CHEF ACCESS CONTROL"
-    banner "knife acl remove OBJECT_TYPE OBJECT_NAME PERM ACTOR_TYPE ACTOR_NAME"
+    banner "knife acl remove OBJECT_TYPE OBJECT_NAME PERMS [client|group|user] ACTOR_NAME"
 
-    attr_reader :object_type, :object_name, :perm, :actor_type, :actor_name
+    attr_reader :object_type, :object_name, :perms, :actor_type, :actor_name
 
     deps do
       include OpscodeAcl::AclBase
     end
 
     def run
-      @object_type, @object_name, @perm, @actor_type, @actor_name = name_args
+      @object_type, @object_name, @perms, @actor_type, @actor_name = name_args
 
       if name_args.length < 5
         show_usage
-        ui.fatal "You must specify the object_type, object_name,  perm, actor type (client or group), and actor name"
+        ui.fatal "You must specify the object_type, object_name, perms, actor type (client, group or user), and actor name"
         exit 1
       end
 
       validate_all_params!
-      ace = get_ace(object_type, object_name, perm)
 
-      case actor_type
-      when "client"
-        remove_actor_from_ace!(actor_name, ace)
-      when "group"
-        remove_group_from_ace!(actor_name, ace)
-      when "users"
-        # Not Implemented yet, we shouldn't get here.
-      end
-
-      update_ace!(object_type, object_name, perm, ace)
-    end
-
-    def remove_group_from_ace!(name, ace)
-      ace['groups'].delete(name)
+      find_actor(actor_type, actor_name)
```

Comment:
It seems that we are always ignoring the return value of this function.  Are we only ever using it to fail early if the given actor doesn't exist?  If so, you might want to give this a name similar to the other "validate" functions.  Does the ACL API return a sensible error if you pass a non-existent user?  If so, why not use that?